report-testing-framework/report-testing/src/test/java/org/jembi/bahmni/report_testing/cdv_ts_reports/HivtmiIndicator1ByEntrypointsTests.java

```diff
@@ -0,0 +1,89 @@
+package org.jembi.bahmni.report_testing.cdv_ts_reports;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import java.util.Map;
+
+
+import org.jembi.bahmni.report_testing.test_utils.BaseReportTest;
+import org.jembi.bahmni.report_testing.test_utils.models.ConceptEnum;
+import org.jembi.bahmni.report_testing.test_utils.models.TherapeuticLineEnum;
+import org.jembi.bahmni.report_testing.test_utils.models.VisitTypeEnum;
+import org.jembi.bahmni.report_testing.test_utils.models.ReportEnum;
+import org.jembi.bahmni.report_testing.test_utils.models.GenderEnum;
+
+import org.joda.time.LocalDate;
+import org.joda.time.LocalDateTime;
+import org.junit.Test;
+
+
+public class HivtmiIndicator1ByEntrypointsTests extends BaseReportTest {
+    @Test
+    public void shouldCountHivPatientByEntryPoints() throws Exception {
+		int patientId = testDataGenerator.registration.createPatient(
+            GenderEnum.MALE,
+            new LocalDate(2000, 9, 1),
+            "John",
+            "Doe"
+        );
+		int encounterId = testDataGenerator.startVisit(
+            patientId,
+            new LocalDate(2022, 11, 1),
+            VisitTypeEnum.VISIT_TYPE_OPD
+		);
+
+        testDataGenerator.program.enrollPatientIntoHIVProgram(
+			patientId,
+			new LocalDate(2022, 11, 3),
+			ConceptEnum.WHO_STAGE_1,
+			TherapeuticLineEnum.FIRST_LINE,
+			new LocalDate(2022, 11, 3)
+		);
+
+        testDataGenerator.ancInitialForm.recordAtAncEnrolmentHivTestDateAndResult(
```

Comment:
@tresorluz we shouldn't need to update the ANC form. We rather need to populate the HIV test date